### PR TITLE
Update Deployment URL Retrieval Procedure in Airflow REST API

### LINKED ISSUE #1783: Update instruction for Deployment URL in Airflow REST API Doc
Originally reported by @wolfier 

### Context

Customers who want to make requests to the Airflow REST API need to retrieve a Deployment URL (URL for the Airflow UI) that is required in the API call. Our current instructions are here: https://docs.astronomer.io/astro/airflow-api

These steps are ok, but they're manual and unable to be retrieved programmatically (which is how users typically make requests to Airflow API). There's now a better way to get this value with the Astro CLI -- `astro deployment inspect`

### Ask

- Update the section in this doc titled `Retrieve the Deployment URL`: https://docs.astronomer.io/astro/airflow-api
- Replace the steps here with steps to get this value from Astro CLI
- The command is: `astro deployment inspect <deployment-name> --key metadata.webserver_url` (verify that `deployment-name` works here and it doesn't have to be `deployment-id`)
- Clarify in the description of this section that this is the URL for the Airflow UI/Airflow webserver, not the Cloud UI and that the term `Deployment URL` will be used
- As a TIP to this section, include the current (manual) steps to grab this URL manually by going to the Airflow UI (the steps currently in the doc). A user would do this if they don't have access to Astro CLI.
- Add @wolfier as a reviewer on this PR
- 

Reference Slack conversation here: https://astronomer.slack.com/archives/C01UA03TPV1/p1674501203794619

### PR DESCRIPTION
**Pull request by kwcanuck**
Resolves #1783 

I was unable to validate the recommended CLI command as an error was returned when I attempted to use it:

image

### CODE REVIEW

**Review by paolaperaza — CHANGES_REQUESTED** [review]
Per discussion today, let's update this such that:

- The Astro CLI steps are the recommended procedure, and there's no additional H4 there
- At the bottom of those steps, we additional mention that this URL is the same as the Airflow UI up to `/home` (you can phrase this better). This is meant to be clarifying, but not actionable or recommended as a step

**Comment by kwcanuck** [comment]
@paolaperaza I have modified the text. Please review and provide your edits or approval.

File: astro/airflow-api.md [hunk 1]
```diff
@@ -48,19 +49,21 @@ curl --location --request POST "https://auth.astronomer.io/oauth/token" \
...
+The token is only valid for 24 hours. If you need to call the Airflow API only once, you can retrieve a single 24-hour access token at `https://cloud.astronomer.io/token` in the Cloud UI.
 
 If you've configured a [CI/CD process](ci-cd.md) and you want to avoid generating an access token manually, Astronomer recommends that you automate the API request to generate a new access token.
 
 :::
 
 ### Retrieve the Deployment URL
 
-The Deployment URL includes the name of your Organization and a short Deployment ID. For example, a Deployment with an ID `dhbhijp0` that is part of an Organization called `mycompany` would have a Deployment URL of `https://mycompany.astronomer.run/dhbhijp0`.
+In the Astro CLI, run the following command:
```
Comment (jwitz) [inline astro/airflow-api.md:60]:
```suggestion
Run the following command to retrieve a Deployment URL:
```
To provide context you could say "in your terminal", but you're not "in the Astro CLI" when you're running CLI commands.

**Comment by kwcanuck** [comment]
THis is a minor change that has been sitting for a month. Edits can be implemented as needed.

**Pull request merged**